web/backend/src/Server.ts

```diff
@@ -175,78 +176,98 @@ app.get('/api/personal_info', (req, res) => {
   }
 });
 
-function isAuthorized(roles: string[], req: express.Request): boolean {
-  if (!req.session || !req.session.userid) {
-    return false;
-  }
+// function isAuthorized(roles: string[], req: express.Request): boolean {
+// return true;
+// if (!req.session || !req.session.userid) {
+// return false;
+// }
 
-  const { role } = req.session;
+// const { role } = req.session;
 
-  return roles.includes(role as string);
-}
+// return roles.includes(role as string);
+// }
+
+const e = newEnforcer('model.conf', 'policy.csv');
 
 // ---
 // Users role
 // ---
-
 // This call allow a user that is admin to get the list of the people that have
 // a special role (not a voter).
-app.get('/api/user_rights', (req, res) => {
-  if (!isAuthorized(['admin'], req)) {
-    res.status(400).send('Unauthorized - only admins allowed');
-    return;
-  }
-
-  const opts: RangeOptions = {};
-  const users = Array.from(
-    usersDB.getRange(opts).map(({ key, value }) => ({ id: '0', sciper: key, role: value }))
-  );
-  res.json(users);
+app.get('/api/user_rights', async (req, res) => {
+  e.then((enforcer) =>
+    enforcer
+      .enforce(req.session.userid, 'roles', 'list')
+      .then((isOK) => {
+        if (isOK) {
+          const opts: RangeOptions = {};
+          const users = Array.from(
+            usersDB.getRange(opts).map(({ key, value }) => ({ id: '0', sciper: key, role: value }))
+          );
+          res.json(users);
+        } else {
+          res.status(400).send('Unauthorized - only admins allowed');
+        }
+      })
+      .catch((erreur) => console.log('error', erreur))
+  ).catch((erreur1) => console.log('error', erreur1));
```

Comment:
avoid french words in the codebase 😁 
also, you can reuse the same variable name, as it's scoped just to that function.

Reply:
Thanks for the view, it is fixed.